### PR TITLE
1262 - Fix UI Actions/Triggers

### LINKED ISSUE #1262: [bug]: UI says Actions instead of Triggers
### Describe the bug

when there are multiple triggers for the same component, the UI says Actions instead of Triggers
image
image

### Steps To Reproduce

1. create a workflow
2. change the manual trigger to Pipdrive trigger

### Expected behavior

it should say "Triggers"

### Version

latest

### OS

ubuntu

### Browser

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by F1nn-T**
## Description

I changed the client-side "OperationSelect" component so it can, like its parent component "CurrentOperationSelect", differentiate if the parameter "triggerSelect" is true or false. Depending on that, it shows "Actions" or "Triggers" as a label.

Fixes #1262  

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by marko-kriskovic** [comment]
@F1nn-T in the CONTRIBUTING.md, there is a Step-by-step guide to contributing section. Could you please redo step 6? When you send a pull request, github runs "./npm run check" automatically and tells you if everything is correct. You can also run that command locally. We are unable to merge this until github says "./npm run check" runs successfully.

File: client/src/pages/platform/workflow-editor/components/CurrentOperationSelect.tsx
Comment (kresimir-coko) [inline client/src/pages/platform/workflow-editor/components/CurrentOperationSelect.tsx:1]: @F1nn-T functionality-wise, the code is good, formatting-wise there are problems. 

Those should be fixed if you run `npm run format` inside the `client` folder. 
You also need to run `npm run typeCheck` to check for TypeScript errors - there is 1 in your code - the `triggerSelect` type is not specified under `CurrentOperationSelectProps`

**Review by kresimir-coko — CHANGES_REQUESTED** [review]

File: client/src/pages/platform/workflow-editor/components/CurrentOperationSelect.tsx
Comment (F1nn-T) [reply client/src/pages/platform/workflow-editor/components/CurrentOperationSelect.tsx:1]: Hey @kresimir-coko , maybe I'm blind :), but the type is defined under the props I cannot see the error sorry

**Comment by igorbeslic** [comment]
@F1nn-T thank you for your contribution. I'll merge your commits. There is limitation in GitHub Actions that prevents us to completely test code after your change.
One important note, please when ever you send code please always do rebasing against bytechef origin repository master branch (git pull --rebase origin master) to avoid merge commits like this [12cf780](https://github.com/bytechefhq/bytechef/pull/1272/commits/12cf78047f530b4dfbf4d32947b2b8ece672fb12).

This time I'll remove it for you.

Thank you for contribution and welcome to the team.

**Pull request merged**